src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSPathogenReferenceTaxonProperties.java

```diff
@@ -11,10 +14,34 @@
     public String name = null;
     public String rank = null;
     public String parentTaxId = null;
-    public long length = 0;
-    public final Set<String> accessions;
+    private long length = 0;
```

Comment:
It appears that way but I sometimes set them directly, i.e. taxonProperties.name = "blah"